hack/make-rules/test-cmd-util.sh

```diff
@@ -4669,3 +4669,257 @@ runTests() {
     exit 1
   fi
 }
+
+run_initializer_tests() {
+  set -o nounset
+  set -o errexit
+
+  create_and_use_new_namespace
+  kube::log::status "Testing --include-uninitialized"
+  ### Create the Initializer
+  kubectl create -f hack/testdata/initializer-configuration.yaml 2>&1 "${kube_flags[@]}"
+  kube::test::get_object_assert initializerconfiguration "{{range.items}}{{$id_field}}{{end}}" 'test-initializer'
+
+  ### Create a deployment
+  kubectl create -f hack/testdata/initializer-deployments.yaml 2>&1 "${kube_flags[@]}" || true
```

Comment:
this command will block for 30 seconds. I'm not sure what should be done for this case, but use something like `timeout 5 kubectl ...  | true`